src/data/events.js

```diff
@@ -1,10 +1,16 @@
 import { writable } from 'svelte/store'
-import { getEvents } from './gqlQueries'
+import { getEvents, joinEvent } from './gqlQueries'
 import { onClear } from './storage'
 
 export const events = writable([])
 export const loading = writable(false)
 
+export async function addToMyEvents(eventId) {
```

Comment:
this name feels a little like a UI concept leaking into the "mid-tier"...mind you, I don't have strong feelings about this but my preference would be to export `join` to consumers, i.e., the UI componentry in this case.

Reply:
Hmm, good point